### PR TITLE
If the search input must not be blank, we should notify users

### PR DESCRIPTION
**Pull request by zhaixiaoxiang**

### CODE REVIEW

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/AccessesController.java [hunk 1]
```diff
@@ -66,8 +66,10 @@ public AccessesController(RouteService routeService, ProviderService providerSer
         AccessDTO accessDTO;
         if (StringUtils.isNotBlank(application)) {
             accessDTO = routeService.findAccess(application);
-        } else {
+        } else if (StringUtils.isNotBlank(service)) {
```
Comment (nzomkxia) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/AccessesController.java]: this is not needed

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/LoadBalanceController.java [hunk 2]
```diff
@@ -92,8 +92,10 @@ public boolean updateLoadbalance(@PathVariable String id, @RequestBody Balancing
         BalancingDTO balancingDTO;
         if (StringUtils.isNotBlank(application)) {
             balancingDTO = overrideService.findBalance(application);
-        } else {
+        } else if (StringUtils.isNotBlank(service)) {
             balancingDTO = overrideService.findBalance(service);
+        } else {
+            throw new ParamValidationException("Either Service or application is required.");
         }
```
Comment (nzomkxia) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/LoadBalanceController.java]: this is not needed

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/OverridesController.java [hunk 3]
```diff
@@ -87,6 +87,8 @@ public boolean updateOverride(@PathVariable String id, @RequestBody DynamicConfi
             override = overrideService.findOverride(service);
         } else if(StringUtils.isNotEmpty(application)){
```
Comment (nzomkxia) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/OverridesController.java]: `notBlank` is better

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/WeightController.java [hunk 4]
```diff
@@ -89,8 +89,10 @@ public boolean updateWeight(@PathVariable String id, @RequestBody WeightDTO weig
         WeightDTO weightDTO;
         if (StringUtils.isNotBlank(application)) {
             weightDTO = overrideService.findWeight(application);
-        } else {
+        } else if (StringUtils.isNotBlank(service)) {
             weightDTO = overrideService.findWeight(service);
+        } else {
+            throw new ParamValidationException("Either Service or application is required.");
         }
```
Comment (nzomkxia) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/WeightController.java]: this is not needed

File: dubbo-admin-ui/src/components/governance/AccessControl.vue [hunk 5]
```diff
@@ -537,7 +541,9 @@ export default {
       this.filter = query['application']
       this.selected = 1
     }
-    this.search()
+    if (this.filter !== null) {
+      this.search()
+    }
   },
   components: {
     Breadcrumb,
```
Comment (nzomkxia) [inline dubbo-admin-ui/src/components/governance/AccessControl.vue]: there's no need to add a new method, you can check `this.filter` in `search()`

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/WeightController.java (hunk 4 above)
Comment (zhaixiaoxiang) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/WeightController.java]: ok

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/OverridesController.java (hunk 3 above)
Comment (zhaixiaoxiang) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/OverridesController.java]: ok

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/LoadBalanceController.java (hunk 2 above)
Comment (zhaixiaoxiang) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/LoadBalanceController.java]: ok

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/AccessesController.java (hunk 1 above)
Comment (zhaixiaoxiang) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/AccessesController.java]: ok

File: dubbo-admin-ui/src/components/governance/AccessControl.vue (hunk 5 above)
Comment (zhaixiaoxiang) [reply dubbo-admin-ui/src/components/governance/AccessControl.vue]: I will notify error msg if this.filter === null in this.search(), but here in mounted(), I just need not to call this.search(), other than notify, so I think this modification is ok

**Comment by zhaixiaoxiang** [comment]
@nzomkxia I have modified your reviews.

**Pull request merged**